--- a/Makefile.in
+++ b/Makefile.in
@@ -339,11 +339,6 @@
 GCCPKGVER :=
 endif
 
-$(srcdir)/%/.git:
-	cd $(srcdir) && \
-	flock `git rev-parse --git-dir`/config git submodule init $(dir $@) && \
-	flock `git rev-parse --git-dir`/config git submodule update --progress --depth 1 $(dir $@)
-
 stamps/install-host-gcc: $(GCC_SRCDIR) $(GCC_SRC_GIT)
 	if test -f $</contrib/download_prerequisites && test "@NEED_GCC_EXTERNAL_LIBRARIES@" = "true"; then cd $< && ./contrib/download_prerequisites; fi
 	rm -rf $@ $(notdir $@)
